pkg/storaged/anaconda.jsx

```diff
@@ -30,19 +30,57 @@
     return null;
 };
 
+function parse_parent_from_options(options) {
+    const parent_match = options.match(/x-parent=(?<parent>[\w\\-]+)/);
+    if (parent_match) {
+        return parent_match.groups.parent;
+    } else
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19924-20240207-125230-6cf61ca9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)